CONTRIBUTING.md

```diff
@@ -2,9 +2,11 @@
 
 We welcome any and all contributions to Kedro, at whatever level you can manage. For example, you could:
 
-- [Join the community](#join-the-community-and-help-it-grow)
-- [Contribute to the project](#contribute-to-the-project)
-- [Join our Technical Steering Committee](#join-our-technical-steering-committee)
+- [Introduction](#introduction)
+  - [Code of conduct](#code-of-conduct)
+  - [Join the community](#join-the-community)
+  - [Contribute to the project](#contribute-to-the-project)
+  - [Join our Technical Steering Committee](#join-our-technical-steering-committee)
```

Comment:
What's the reason to add the Introduction header?

Reply:
The introduction header was already on the original file. Do you think we could remove it? Or maybe replace it with a different title.

Reply:
I am referring to the table of content - I don't think the Introduction header is necessary but maybe Jo will have comment on this.

Reply:
I don't think it needs any further changes at this point, personally I think it's easy to read and understand.

I've also moved the other contributing guideline pages to the wiki. If everything is OK with them, I can open another PR removing the reduntant info from the docs.

Reply:
I took a look at those changes and they're great. There are a few link changes to do because the docs have moved and I think a bit of reorganisation in the wiki itself to group the pages into definite sections. I'll take a look later today at this and we can pair on it if needed.